src/reporter.js

```diff
@@ -180,7 +193,37 @@ Reporter.prototype = {
 
   flush: function () {
     // This runs only once and only at the very end when we're shutting down all the reporters
-    logger.log("Admiral2 reporter shutting down.");
+    var deferred = Q.defer();
+    var self = this;
+
+    // finalize test run status
+    self.runOptions.result = "pass";
+
+    _.forEach(self.results, function(value){
+      _.forEach(value, function(innerValue){
+        if(innerValue.status === "fail"){
+          self.runOptions.result = "fail";
+        }
+      });
+    });
+
+    fetch(ADMIRAL_URL + "api/project/" + ADMIRAL_PROJECT + "/" + ADMIRAL_PHASE + "/run/" + ADMIRAL_RUN + "/finish", {
+      headers: { "Content-Type": "application/json" },
+      method: "POST",
+      body: JSON.stringify(self.runOptions)
+    })
+    .then(function (res) {
+      var reportURL = ADMIRAL_URL + "run/" + ADMIRAL_RUN;
+      logger.log("Visual report matrix available at: " + reportURL);
```

Comment:
Let's update this message with something like `Visualized test suite results available at:`